cli/tauri.js/api/fs/index.js

```diff
@@ -0,0 +1,139 @@
+import tauri from './tauri'
+import { Dir } from './dir'
+
+/**
+ * reads a file as text
+ *
+ * @param {string} filePath path to the file
+ * @param {object} [options] configuration object
+ * @param {BaseDirectory} [options.dir] base directory
+ * @return {Promise<string>}
+ */
+function readTextFile (filePath, options = {}) {
+  return tauri.readTextFile(filePath, options)
+}
+
+/**
+ * reads a file as binary
+ *
+ * @param {string} filePath path to the file
+ * @param {object} [options] configuration object
+ * @param {BaseDirectory} [options.dir] base directory
+ * @return {Promise<int[]>}
+ */
+function readBinaryFile (filePath, options = {}) {
+  return tauri.readBinaryFile(filePath, options)
+}
+
+/**
+ * writes a text file
+ *
+ * @param {object} file
+ * @param {string} file.path path of the file
+ * @param {string} file.contents contents of the file
+ * @param {object} [options] configuration object
```

Comment:
Do we need an encoding? or do we assume UTF-8?

Reply:
we just convert the string to a byte array, but we should probably have an endpoint to store binary files too